frontend/src/ui/Blocks/Series.tsx

```diff
@@ -261,32 +266,50 @@ const SeriesBlockContainer: React.FC<SeriesBlockContainerProps> = (
             borderRadius: 10,
             ...isDark && darkModeBoxShadow,
         }}>
-            <div css={{
-                display: "flex",
-                [screenWidthAtMost(VIDEO_GRID_BREAKPOINT)]: {
-                    flexWrap: "wrap",
-                },
-            }}>
-                {title && <h2 css={{
-                    display: "inline-block",
-                    padding: "8px 12px",
-                    color: isDark ? COLORS.neutral90 : COLORS.neutral80,
-                    fontSize: 20,
-                    lineHeight: 1.3,
-                }}>{title}</h2>}
-                {showViewOptions && <div css={{
+            <>
+                <div css={{
                     display: "flex",
-                    alignItems: "center",
-                    alignSelf: "flex-start",
-                    marginLeft: "auto",
-                    fontSize: 14,
-                    gap: 16,
-                    padding: 5,
+                    justifyContent: title ? "space-between" : "flex-end",
+                    ...title && description && { flexDirection: "column" },
```

Comment:
I think the conditions can be removed. If there is no `title` then that div only has one flex-child. So the direction and `justify-content` make no difference. Maybe double check but pretty sure.

```
justifyContent: "space-between"
flexDirection: "column",
```

Reply:
Oh you are right about the `justifyContent`. 
`flexDirection` is a little more involved unfortunately, with the existing structure at least (child one is the title, child two is either the buttons or description + buttons):
- case 1 (neither title nor description is shown): one child, `flexDirection` doesn't matter
- case 2 (title shown but no description): two children,  `flexDirection` needs to be `row`
- case 3 (description is shown but no title): one child, `flexDirection` doesn't matter
- case 4 (title and description are shown): two children,  `flexDirection` needs to be `column`

So this needs some condition, and I think the current one is the simplest way to do it. Unless I'm overthinking this.

Reply:
Oh yep, my bad.